javascript-carousel/main.js

```diff
@@ -14,9 +14,6 @@ function handlePrevImage(event) {
     dotMarker();
   } else {
     counter = 5;
-    counter--;
-    imageAttr();
-    dotMarker();
```

Comment:
Does this work after removing these entirely like this? I meant that you should have that code below the conditional to run in both situations, this looks like it will only advance the image if `counter > 0`

Reply:
It actually had make the interval 6 seconds at the end before it looped over again. I think this revision addresses the issue, thanks for the feedback!